proto/beacon/rpc/v1/services.proto

```diff
@@ -22,23 +22,28 @@ service ProposerService {
 }
 
 service ValidatorService {
+    // These endpoints can be called on demand in the future
+    // by some web3 API for users to conveniently know their assignment.
     rpc ValidatorShardID(PublicKey) returns (ShardIDResponse);
     rpc ValidatorIndex(PublicKey) returns (IndexResponse);
     rpc ValidatorSlot(PublicKey) returns (SlotResponse);
+    // This endpoint is called by all validator clients to watch for assignments
+    // for a subset of public keys in the active validator set.
+    rpc ValidatorAssignment(ValidatorAssignmentRequest) returns(stream ValidatorAssignmentResponse);
 }
 
 message GenesisTimeAndStateResponse {
-  google.protobuf.Timestamp genesis_timestamp = 1;
-  ethereum.beacon.p2p.v1.CrystallizedState latest_crystallized_state = 2;
+    google.protobuf.Timestamp genesis_timestamp = 1;
+    ethereum.beacon.p2p.v1.CrystallizedState latest_crystallized_state = 2;
 }
 
 message ProposeRequest {
```

Comment:
Late to the party, we can address this in the next PR.
We don't need `attestation_bitmask` and `attestation_aggregate_sig` inside a block. What we need is aggregated attestations so like `repeated AggregatedAttestation attestations`